_posts/2022-10-15-1.16-announcement.md

```diff
@@ -0,0 +1,402 @@
+- [USING JAR](https://nightlies.apache.org/flink/flink-docs-release-1.16/docs/dev/table/sql/create/#create-function) 
+  supports dynamic loading of UDF jar to help platform developers to easily manage UDF.
+- [CREATE TABLE AS SELECT](https://nightlies.apache.org/flink/flink-docs-release-1.16/docs/dev/table/sql/create/#as-select_statement) (CTAS) 
+  supports users to create new tables based on existing tables and queries.
+- [ANALYZE TABLE](https://nightlies.apache.org/flink/flink-docs-release-1.16/zh/docs/dev/table/sql/analyze) 
+  supports users to manually generate table statistics so that the optimizer could 
+  generate better execution plans.
+
+## Cache in DataStream for Interactive Programming
+
+Supports caching the result of a transformation via [DataStream#cache](https://nightlies.apache.org/flink/flink-docs-release-1.16/docs/dev/datastream/operators/overview/#cache). 
+The cached intermediate result is generated lazily at the first time the intermediate result is 
+computed so that the result can be reused by later jobs. If the cache is lost, 
+it will be recomputed using the original transformations. Currently only batch mode is supported. 
+This feature is very useful for ML and interactive programming in Python.
+
+## History Server & Completed Jobs Information Enhancement
+
+We have enhanced the experiences of viewing completed jobs’ information in this release.
+- JobManager / HistoryServer WebUI now provides detailed execution time metrics, 
+  including duration tasks spent in each execution state and the accumulated 
+  busy / idle / back-pressured time during running.
+- JobManager / HistoryServer WebUI now provides aggregation of major SubTask metrics, 
+  grouped by Task or TaskManager.
+- JobManager / HistoryServer WebUI now provides more environmental information, 
+  including environment variables, JVM options and classpath.
+- HistoryServer now supports browsing logs[6] from external log archiving services.
+
+## Protobuf format
+
+Flink now supports the Protocol Buffers (Protobuf) format. This allows you to use this format directly 
+in your Table API or SQL applications.
+
+## Introduce configurable RateLimitingStrategy for Async Sink
+
+The Async Sink was implemented in 1.15 to allow users to easily implement 
+their own custom asynchronous sinks. We have now extended it to support 
+a configurable RateLimitingStrategy. This means sink implementers can now customize 
+how their Async Sink behaves when requests fail, depending on the specific sink. 
+If no RateLimitingStrategy is specified, it will default to the current default of AIMDRateLimitingStrategy.
+
+# Upgrade Notes
+
+We aim to make upgrades as smooth as possible, but some of the changes require users to 
+adjust some parts of the program when upgrading to Apache Flink 1.16. Please take a look at 
+the release notes for a list of adjustments to make and issues to check during upgrades.
```

Comment:
Have we added the list of all necessary adjustments in the release notes?

Reply:
No, there is another blog, see https://github.com/apache/flink/pull/20859